src/main/java/seedu/address/MainApp.java

```diff
@@ -73,10 +78,12 @@ public void init() throws Exception {
      * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
-        logger.info("Using data file : " + storage.getAddressBookFilePath());
+        logger.info("Using data file : " + storage.getAddressBookFilePath() + storage.getEventBookFilePath());
 
         Optional<ReadOnlyAddressBook> addressBookOptional;
+        Optional<ReadOnlyEventBook> eventBookOptional;
         ReadOnlyAddressBook initialData;
+        ReadOnlyEventBook initialData2;
```

Comment:
I'll do a proper review later, for now change these names so that they are more meaningful. 
Etc initialAddressBookData

Reply:
The rest of it LGTM

Reply:
Solved thanks